fastlane/Fastfile

```diff
@@ -110,6 +110,28 @@ lane :release do |options|
   UI.success("Successfully started release #{version_number}")
 end
 
+lane :merge_release_to_main do
+  ensure_git_status_clean
+  sh('git checkout main')
+  sh('git pull')
+  release_branches = sh(command: 'git branch -a', log: false).delete(' ').split("\n").grep(%r(origin/.*release/))
+  UI.user_error!("Expected 1 release branch, found #{release_branches.size}") if release_branches.size != 1
+  sh("git merge #{release_branches.first} --ff-only") # ff-only is important. Read: https://notion.so/iOS-Branching-Strategy-37c10127dc26493e937769d44b1d6d9a
```

Comment:
What is this part actually doing? Maybe you can provide a comment? How does it know it is the most recent release?

Reply:
Actually, it does not know anything about release branches apart from their cound. It searches for all `release` branches in `origin` and:
- Raises a error if there are more than one branch
- Picks the branch name of there is only one branch
 
As per our workflow we always have only one release branch, I thought it is safe to do it this way.

If you think it's not the ideal option, I can provide a `prompt` if there is more than one branch so the release engineer could pick the correct one manually. Wdyt?

Reply:
Done. Check it out 🙂